packages/workflow/src/Expression.ts

```diff
@@ -298,6 +298,15 @@ export class Expression {
 					throw error;
 				}
 			}
+
+			// temporary divergence until breaking change
```

Comment:
this comment can be more clear.. what's diverging.. what's breaking.. why temporary.. confusing without Slack context